include/roboteam_ai/stp/evaluations/game_states/TimeOutGameStateEvaluation.h

```diff
@@ -0,0 +1,23 @@
+//
+// Created by jordi on 28-04-20.
+//
+
+#ifndef RTT_TIMEOUTGAMESTATEEVALUATION_H
+#define RTT_TIMEOUTGAMESTATEEVALUATION_H
+
+#include "stp/evaluations/BaseEvaluation.h"
+
+namespace rtt::ai::stp::evaluation {
+
+/**
+ * Invariant for the game state timeout
+ */
+class TimeOutGameStateEvaluation : public BaseEvaluation {
+   public:
+    [[nodiscard]] uint8_t metricCheck(world::view::WorldDataView world, const world::Field* field) const noexcept override;
+
+    const char* getName() override { return "gs::TimeOut"; }
```

Comment:
This naming could be dangerous if getName() is used to compare the different classes with eachother, so please check if that doesn't happen.

Reply:
I'll look into it

Reply:
https://github.com/RoboTeamTwente/roboteam_ai/blob/ScoreImplement/src/stp/PlayChecker.cpp#L30
https://github.com/RoboTeamTwente/roboteam_ai/blob/ScoreImplement/src/ApplicationManager.cpp#L177

Currently only used to hardcode plays. It compares the strings.
I dont get why they did this, it would be easier to just get the play from the vector.
Ill add a TODO to remove this function.